boltons/timeutils.py

```diff
@@ -11,12 +11,41 @@
 
 import re
 import bisect
+import math
+import time
 import datetime
 from datetime import timedelta
 
+# optional dependency: monotonic time
+try:
+    # python3.3 added monotonic time to stdlib
+    monotonic = time.monotonic
+except AttributeError:
+    try:
+        from importlib import import_module
+    except ImportError:
+        monotonic = None # py2.6
+    else:
+        # lib name, function that gets value given module (in order of preference)
+        _monotonic_libs = [
```

Comment:
That's a good point.
I might keep the >= python3.3 `time.monotonic()` check though, and have it always use that if available.
Then python3 and python2 users both get a sane, expected behaviour.